### ISSUE TITLE
let*

### ISSUE THREAD

**Issue by vd1** [jkrivine/tl_interpreter#5]
I do not understand when to use `let`, when to use `let*`; would be nice to update `example1.ml` with some additional explanations.

**Comment by adhusson** [comment]
I added some explanations in [`README.md`](https://github.com/jkrivine/tl_interpreter/blob/master/README.md) in the commit eaa2b5dcc22f3789, let me know if it helps!

**Comment by vd1** [comment]
> The infix version of `bind` is `>>=`; it is convenient for chaining commands:
> ```                                                                                       
> (get_balance userA) >>= (fun balance -> <do something with userA's balance>)              
> ```                                                                                       
> where (for illustration purposes) `balance user (state,context)                           
> = state.balances[user]`

did you mean `get_balance (state,context) = state.balance[user]`?

**Comment by adhusson** [comment]
Yes I mangled the function name. Also it's actually `get_balance user (state,context) = (Ok state.balances[user], state)` (fixed in 0b2c92f).
